contribs/drt/src/test/java/org/matsim/contrib/drt/util/DrtEventsReadersTest.java

```diff
@@ -65,7 +65,7 @@
 
 	//standard dvrp events are tested in DvrpEventsReadersTest
 	private final List<Event> drtEvents = List.of(
-			new DrtRequestSubmittedEvent(0, mode, request, person, link1, link2, 111, 222),//
+			new DrtRequestSubmittedEvent(0, mode, request, person, link1, link2, 111, 222, Double.NaN, Double.NaN),//
```

Comment:
For testing event writing/reading I would use something else than `NaN` and perhaps different values for each field (to make sure they are not confused).